python_modules/libraries/dagster-databricks/dagster_databricks/components/databricks_asset_bundle/component.py

```diff
@@ -139,11 +146,48 @@ class DatabricksAssetBundleComponent(Component, Resolvable):
             ],
         ),
     ] = None
+    asset_specs_by_task_key: Optional[dict[str, list[ResolvedAssetSpec]]] = None
```

Comment:
For the demo implementation I originally started with an `asset_specs` attribute, but was suggested to instead simply do `assets`, as this may be more intuitive for end users. We may want to consider simplifying this to:

* `asset_by_task_key`

Or alternatively something like:

* `asset_task_mapping`

Reply:
Makes sense - updated to assets_by_task_key in [b32173a](https://github.com/dagster-io/dagster/pull/32214/commits/b32173a0a0860832fd07d09e064eb4079905a172)